### PR TITLE
composition docs fix

### PR DESCRIPTION
**Pull request by prasek**
### Description of your changes

Minor composition docs fix

I have:

- [x] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: docs/introduction/composition.md [hunk 1]
```diff
@@ -21,15 +21,15 @@ their application operators, who may then declare that their application
 requires that kind of resource.
 
 Composition can be used to build a catalogue of kinds and configuration classes
-of infrastructure that fit the needs and opinions of your organisation. As an
+of infrastructure that fit the needs and opinions of your organization. As an
```
Comment (hasheddan) [inline docs/introduction/composition.md:24]: This change is fine with me, but both are correct :) https://writingexplained.org/organisation-vs-organization-difference

Comment (prasek) [reply docs/introduction/composition.md:24]: @hasheddan interesting, didn't realize the British allowed both forms. :)

Comment (hasheddan) [reply docs/introduction/composition.md:24]: Guessing Australia does as well cc @negz :)

Comment (negz) [reply docs/introduction/composition.md:24]: Working as intended. 🇦🇺

Comment (prasek) [reply docs/introduction/composition.md:24]: https://www.nngroup.com/articles/american-vs-british-english-for-web/

**Pull request closed without merging**